playbooks/logging.yml

```diff
@@ -1,42 +1,82 @@
 ---
-- name: Install logging and dependencies
+- name: cleanup old configs and install logstash-forwarder
   hosts: all
+  tasks:
+    - include: ../roles/cleanup/tasks/logging.yml
+    - include: ../roles/common/tasks/bbg-repo.yml
   roles:
-    - ../roles/common
     - ../roles/logging
 
-- name: keystone code and config
-  hosts: controller:db
+- name: keystone logging
+  hosts: controller
+  vars_files:
+    - ../roles/keystone/defaults/main.yml
   roles:
-    - ../roles/keystone-common
+    - role: ../roles/logging-config
+      service: keystone
+      logdata: "{{ keystone.logs }}"
 
-- name: neutron code and config
-  hosts: controller:db:compute:network
+- name: glance logging
+  hosts: controller
   vars_files:
-    - ../roles/rabbitmq/defaults/main.yml
+    - ../roles/glance/defaults/main.yml
   roles:
-    - ../roles/neutron-common
+    - role: ../roles/logging-config
+      service: glance
+      logdata: "{{ glance.logs }}"
 
-- name: nova code and config
-  hosts: controller:db:compute
+- name: nova logging
+  hosts: controller:compute
   vars_files:
-    - ../roles/rabbitmq/defaults/main.yml
+    - ../roles/nova-common/defaults/main.yml
   roles:
-    - ../roles/nova-common
+    - role: ../roles/logging-config
+      service: nova
+      logdata: "{{ nova.logs }}"
 
-- name: swift common code and config
-  hosts: swiftnode
+- name: cinder logging
+  hosts: controller:compute
+  vars_files:
+    - ../roles/cinder-common/defaults/main.yml
   roles:
-    - ../roles/swift-common
+    - role: ../roles/logging-config
+      service: cinder
+      logdata: "{{ cinder.logs }}"
 
-- name: glance code and config
-  hosts: controller:db
+- name: neutron logging
+  hosts: network:compute
+  vars_files:
+    - ../roles/neutron-common/defaults/main.yml
   roles:
-    - ../roles/glance-common
+    - role: ../roles/logging-config
+      service: neutron
+      logdata: "{{ neutron.logs }}"
 
-- name: cinder code and config
-  hosts: controller:db:compute
-  vars_files:
-    - ../roles/rabbitmq/defaults/main.yml
+
+# TODO: These roles are missing logging configs!!!
```

Comment:
Ah, good point. I don't believe will be targeted to have full log forwarding in 1.0.x, at least not at this point. I will remove this!